README.md

```diff
@@ -294,3 +294,28 @@ change_processor:
         # Terraform workspaces list
         workspace: ["my-workspace-no1"]       
 ```
+
+## Single run mode
+
+Allows to run cloudimized only to scan given resource and dump them into text files, without performing and additional
+functions (no Git, Terraform, Slack, Jira interaction and no GCP logs lookup).
+
+### Running
+
+```yaml
+python3 main.py --singlerun/-s <RESOURCE_NAME> --output/-o {yaml, csv}
```

Comment:
`python3 main.py` -> `cloudimized`?

Reply:
Thanks - fixed in https://github.com/egnyte/cloudimized/commit/aa1c272bd79ff17791410a62f52674e8bbed34fb